doc/api/next_api_changes/behaviour.rst

```diff
@@ -155,3 +155,9 @@ support for it will be dropped in a future Matplotlib release.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Previously, keyword arguments were silently ignored when no positional
 arguments were given.
+
+
+`get_xticklabels` and `get_yticklabels` now returns plain list
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Previously, get_xticklabels and get_yticklabels returns silent_list.
```

Comment:
Can enumerate all of the methods that will be changed?  The audience for this will not have the context you current have.  What are the consequences of this change? Who is likely to be affected?  What can they do to get the old behavior back?

Reply:
I have two questions in my mind. 1st next_api_changes are the changes in the upcoming release which has not been shipped to market yet. Please correct me if I am wrong.
2nd: behaviour.rst is meant for knowing the changes across releases. Correct me if I am wrong. 
thanks